client/post-editor/media-modal/test/index.jsx

```diff
@@ -300,8 +292,8 @@ describe( 'EditorMediaModal', () => {
 			// EditorMediaModal will generate transient ID for the media selected
 			// by using uniqueId, which increments its value within the same session.
 			const transientItems = [
-				Object.assign( {}, DUMMY_MEDIA[ 0 ], { ID: 'media-1', transient: true } ),
-				Object.assign( {}, DUMMY_MEDIA[ 1 ], { ID: 'media-2', transient: true } ),
+				Object.assign( {}, DUMMY_MEDIA[ 0 ], { ID: 'media-fake-uuid', transient: true } ),
+				Object.assign( {}, DUMMY_MEDIA[ 1 ], { ID: 'media-fake-uuid', transient: true } ),
```

Comment:
To keep the original intent, should keep `-1` and `-2` suffixes?

Reply:
We could, but I'd have to complicate mocking a global and pollute other tests. I could cache the global, overwrite the mock, then reset the global. But is it worth it?

Reply:
It could be worth it in some cases, to differentiate between the various IDs. Theoretically, we could at some point introduce bugs that duplicate IDs and we wouldn't catch them.